src/model/writer.js

```diff
@@ -802,14 +806,19 @@ export default class Writer {
 	 *
 	 *		addMarker( markerName, { range, usingOperation: true } );
 	 *
+	 * Create marker that doesn't affects the data model:
+	 *
+	 *		addMarker( markerName, { range, usingOperation: false, affectData: false } );
+	 *
 	 * Note: For efficiency reasons, it's best to create and keep as little markers as possible.
 	 *
 	 * @see module:engine/model/markercollection~Marker
 	 * @param {String} name Name of a marker to create - must be unique.
 	 * @param {Object} options
-	 * @param {Boolean} options.usingOperation Flag indicated whether the marker should be added by MarkerOperation.
+	 * @param {Boolean} options.usingOperation Flag indicating that the marker should be added by MarkerOperation.
 	 * See {@link module:engine/model/markercollection~Marker#managedUsingOperations}.
 	 * @param {module:engine/model/range~Range} options.range Marker range.
+	 * @param {Boolean} [options.affectsData=true] Flag indicating that the marker changes the data model.
```

Comment:
Why does it default to `true`?

If we want it to default to `true`, we should think about changing its name so it can default to `false`.

Reply:
I remember that we couldn't think of a better name with PJ. The opposite to `affectsData` could be `transparent` (but that name is reserved) or smth like `notAffectingData`. If you have some proposals, I'm open to them :)

I agree that parameters that default to false are better, but on the other hand, descriptive parameter names are also important.

Reply:
> Why does it default to true?

This question is still open ;)